Makefile

```diff
@@ -96,7 +96,7 @@ clean:
 	@$(MAKE) -C $(ZWRAPDIR) $@ > $(VOID)
 	@$(MAKE) -C examples/ $@ > $(VOID)
 	@$(MAKE) -C contrib/gen_html $@ > $(VOID)
-	@$(RM) zstd$(EXT) zstdmt$(EXT) tmp*
+	@$(RM) -rf zstd$(EXT) zstdmt$(EXT) tmp* lz4
```

Comment:
`-f` is implied by `$(RM)` : https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html
`-r` is not, and is only useful for directory.

For clarity, I would prefer to separate deleting files from deleting directories.
Hence, it would keep existing line, and add another one : 
`@$(RM) -r lz4`
This makes it easier to understand that `lz4` is a directory in this context.